### ISSUE TITLE
Mystical World Duplicate/Redundant Ores

### ISSUE THREAD

**Issue by kythorn** [NillerMedDild/Enigmatica2#586]
**Modpack version**
1.74

**Issue**
Mystical World Duplicate Ores

**Logs**
N/A

The mystical worlds config file is setup to spawn copper and silver. Setting the "PerChunk" variables should resolve this. I believe this came up and was resolved before, but it's back now. 

Amethyst may or may not be redundant, I'm not sure offhand.

**Comment by NillerMedDild** [comment]
Fixed again, thanks for letting me know :)

**Comment by MakiabelMFE** [comment]
I'm posting this issue here since it's related to the OP and since it's already been mark as completed there's no real point in creating a new one.
 I can't get this to work, I've changed every single instance of "ore enabled" in the mysticalworld config file to false, changed every single value related to ore gen to 0. I cannot get this dumb ore out of my world, it's already generated like that.
I tried MCEdit beta 14 or whatever, tried changing the copper and silver ore to air and stone, both times it changes it to a turbine power block or something?! whatever, MCEdit unified then, nope, selection only, can't change all chunks so uh, yeah.
Tried the WorldUtils mod, crashes during the task of replacing the block, awesome. WorldEdit? apparently also selection of 32k blocks or less.

I'm about to just remove the mod entirely, anyone got a clue as to what else can I do? or is the mod not important enough that I can just remove it?
